tests/test_continuous_futures.py

```diff
@@ -49,27 +49,37 @@ def make_root_symbols_info(self):
     @classmethod
     def make_futures_info(self):
         return DataFrame({
-            'symbol': ['FOF', 'FOG', 'FOH'],
-            'root_symbol': ['FO', 'FO', 'FO'],
-            'asset_name': ['Foo'] * 3,
+            'symbol': ['FO16F', 'FO16G', 'FO16H', 'FO16J', 'FO22F'],
```

Comment:
Our symbols would be formatted like 'FOF16', etc., might be worth changing to avoid confusion